core/src/mapping.rs

```diff
@@ -79,6 +103,58 @@ impl<K: OdraType + Hash, V: OdraType> Instance for Mapping<K, V> {
     }
 }
 
+#[cfg(feature = "mock-vm")]
```

Comment:
Why Mapping needs serialization?

Reply:
Modules (including Mapping and Variable) are no longer serialized